eXo/Sources/Tool.swift

```diff
@@ -29,4 +29,27 @@ class Tool {
         }
     }
 
+    //Remove the protocol (http:// or https://) of a URL in string
+    static func stringURLWithoutProtocol (stringURLWithProtocol : String) -> String {
```

Comment:
Implement this as a Swift extension (category) https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Extensions.html of `String`